build-tools/automation/templates/core-tests.yaml

```diff
@@ -173,6 +173,21 @@ steps:
     arguments: -c $(Build.Configuration) tools/java-source-utils/java-source-utils.csproj -t:RunTests
   continueOnError: true
 
+- powershell: >
```

Comment:
I have no idea, I'm just going with @akoeplinger 's suggestion on Discord.

Reply:
`script` would work since the command is the same i.e. it doesn't matter whether cmd or bash is used but I agree that this should probably only execute on Mac.